doc/jsk_pcl_ros/nodes/euclidean_clustering.md

```diff
@@ -48,6 +53,29 @@ sensor_msgs/PointCloud2[] output
 
    The minimum number of the points of one cluster.
 
+* `~multi` (Boolean, default: `false`)
+
+   Flag of multi euclidean clustering. If `~multi` is `true`, synchronized `~input` and `~input/cluster_indices` are used.
+
+* `~approximate_sync` (Boolean, default: `False`):
+
+   Policy of synchronization, if `false` it synchornizes exactly, else approximately.
+   This value is only valid in case of `~multi` is `true`.
+
+* `~queue_size` (Int, default: `20`):
+
+   Queue size of topic msgs for synchronization.
+
+* `~downsample_enable` (Boolean, default: `false`)
+
+   Flag of VoxelGrid downsampling. If `~downsample_enable` is `true`, `~input` is downsampled.
+
+* `~leaf_size` (Double, default: `0.01`)
+
+   Leaf size of voxel grid downsampling.
+   This value is only valid in case of `~downsample_enable` is `true`.
+
+
```

Comment:
can you add doc for `cluster_filter`?

Reply:
OK. I added.

Reply:
Someone may misunderstand this parameter that this filtering is applied to `~input/cluster_indices`.
Please explain that this filtering is applied after each euclidean clustering.
So even if you set `1: max size`, this node publishes at most [size of `~input/cluster_indices`] clusters if `~multi` is true.